support-frontend/app/services/pricing/DefaultPromotionsService.scala

```diff
@@ -0,0 +1,68 @@
+package services.pricing
+
+import com.amazonaws.services.s3.AmazonS3URI
+import com.gu.aws.AwsS3Client
+import com.gu.support.config.{Stage, TouchPointEnvironments}
+import io.circe._
+import io.circe.parser._
+import io.circe.generic.auto._
+import akka.actor.ActorSystem
+import com.gu.support.catalog.{GuardianWeekly, Product}
+import com.typesafe.scalalogging.LazyLogging
+import services.pricing.DefaultPromotionsService.DefaultPromotions
+
+import java.util.concurrent.atomic.AtomicReference
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.DurationInt
+import scala.util.{Failure, Success, Try}
+
+trait DefaultPromotionsService {
```

Comment:
Bit of a nitpick but I think I'd go singular with the class name so `DefaultPromotionService` I think it is more standard for the code base. Feel free to disagree though!